<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCGeneratorClass</class>
 <widget class="QWidget" name="TCGeneratorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用例生成器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>AETG法</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>源(.txt)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_ori_path">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 255, 100)</string>
          </property>
          <property name="text">
           <string>//路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_choose_ori">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_ori_info">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(120, 120, 150);
color:rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>当前有待测项??项，覆盖深度需小于等于待测项数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>随机尝试次数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_rand_try">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>覆盖深度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_T_way">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>目标(.xlsx)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_dst_path">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 255, 100)</string>
          </property>
          <property name="text">
           <string>//路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_choose_dst">
          <property name="text">
           <string>选择路径</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>用例表名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_TC_name">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 255, 200)</string>
          </property>
          <property name="text">
           <string>TC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_gen_TC">
          <property name="text">
           <string>生成用例</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TCGenerator.qrc"/>
 </resources>
 <connections/>
</ui>
